library/build.gradle

```diff
@@ -64,5 +64,5 @@ dependencies {
 
 // bintray upload scripts
 // https://inthecheesefactory.com/blog/how-to-upload-library-to-jcenter-maven-central-as-dependency/en
-apply from: 'install.gradle'
-apply from: 'bintray.gradle'
\ No newline at end of file
+//apply from: 'install.gradle'
```

Comment:
Because I was not able to compile the code due to these dependency. I am fixing them.